notifications/main.go

```diff
@@ -0,0 +1,38 @@
+package main
```

Comment:
```suggestion
package notifications
```

after you move the main to /lambda/notifications/x.go you can rename the package here